wallet/types/src/utxo_types.rs

```diff
@@ -41,15 +43,18 @@ pub enum UtxoState {
     Abandoned = 1 << 4,
 }
 
-pub fn get_utxo_type(output: &TxOutput) -> UtxoType {
+pub fn get_utxo_type(output: &TxOutput) -> Option<UtxoType> {
```

Comment:
Not every TxOutput represents Utxo. `UtxoType` needs a revamp to represent that concept, so I placed a TODO because I didn't want to go too deep into the wallet